<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="24554" started-at="2017-04-18T08:20:41Z" finished-at="2017-04-18T08:21:05Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="24554" started-at="2017-04-18T08:20:41Z" finished-at="2017-04-18T08:21:05Z">
      <class name="bahaso.testing.android.Voucher_test">
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:bahaso.testing.android.Voucher_test@59494225]" name="beforeMethod" is-config="true" duration-ms="17095" started-at="2017-04-18T15:20:41Z" finished-at="2017-04-18T15:20:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="inputVoucher_is_alphanumeric()[pri:0, instance:bahaso.testing.android.Voucher_test@59494225]" name="inputVoucher_is_alphanumeric" duration-ms="6552" started-at="2017-04-18T15:20:58Z" finished-at="2017-04-18T15:21:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[input voucher code tidak alphanumeric expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: input voucher code tidak alphanumeric expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at bahaso.testing.android.Voucher_test.inputVoucher_is_alphanumeric(Voucher_test.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- inputVoucher_is_alphanumeric -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:bahaso.testing.android.Voucher_test@59494225]" name="afterMethod" is-config="true" duration-ms="893" started-at="2017-04-18T15:21:04Z" finished-at="2017-04-18T15:21:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- bahaso.testing.android.Voucher_test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
